samples/package.json

```diff
@@ -1,20 +1,26 @@
 {
   "name": "nodejs-security-center-samples",
-  "main": "quickstart.js",
   "private": true,
   "engines": {
     "node": ">=8"
   },
   "scripts": {
-    "test": "mocha system-test"
+    "test": "mocha system-test/*.js --timeout 6000000",
+    "check": "gts check",
+    "clean": "gts clean",
+    "fix": "gts fix"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@google-cloud/security-center": "^0.3.0"
+    "@google-cloud/security-center": "^0.3.0",
+    "gts": "^0.9.0",
+    "lint": "^1.1.2"
```

Comment:
do we need `lint` if we're using `gts` for enforcing style?

Reply:
I think this was added automatically but I mis-remembering.  I removed it.